navigation.yaml

```diff
@@ -1809,12 +1809,13 @@ cube:
     - title: Overview
       path: /cube
 
-    - title: Microcerts
-      path: /cube/microcerts
-
-    - title: Study
-      path: /cube/study
-      persist: True
+    # Temporarily disable for closing the beta
+    # - title: Microcerts
+    #  path: /cube/microcerts
+    #
+    # - title: Study
+    #  path: /cube/study
+    #  persist: True
```

Comment:
I wonder if we should just remove it altogether instead. We can always look into git's history to restore it when we need.

Reply:
I went ahead and removed all the commented-out code.